CMakeLists.txt

```diff
@@ -64,23 +69,27 @@ compute_project_args(PROJECT_ARGS LANGUAGES CXX)
 project(${PROJECT_NAME} ${PROJECT_ARGS})
 
 if(BUILD_PYTHON_INTERFACE)
-  add_project_dependency(eigenpy 3.0.0 REQUIRED)
-  add_project_dependency(pinocchio 2.7.0 REQUIRED)
```

Comment:
Pinocchio is a project dependency. Any reason to do this?

Reply:
I understand. Imposing this dependency ensures that users have installed Pinocchio from the user side. This is the only requirement where (I think) we should consider here.